### DISCUSSION TITLE
M2M Examples for docs

### DISCUSSION THREAD

**Discussion by xaitec** [AmbitionEng/django-pghistory#172] [Q&A] [answered]
Hi, I'm fairly new to the project and would like to switch the project I am working on entirely to pghistory. Which means being able to track eventhing currently being tracked. 

I am currently working on getting the m2m fields tracked but cannot seem to find a way to get it to directly ref the main model it belongs to. Is there any way to do this. If so, can an example be added to the docs? 

P.S. If I figure it out I'll submit a PR to the docs

**Comment by wesleykendall** [comment, marked as answer]
https://django-pghistory.readthedocs.io/en/3.4.4/event_tracking/?h=many#many-to-many-fields discusses how to track M2M fields.

A M2M model effectively just has a primary key and two foreign keys to the relationships. If you check out the example in the docs, two trackers have been set up to just track inserts and deletes to this table, allowing you to preserve a history of when M2M relationships are added and deleted.

**Reply by wesleykendall** [reply]
Im going to go ahead and mark this as the answer and close the Q&A. Let me know if my answer was off the mark of the original question!

**Comment by xaitec** [comment]
You can. I probably was trying to do more than the intended use. I'll work
on it.
